### PR TITLE
fix-execute-proposal

### PR DESCRIPTION
**Pull request by Mati0x**
#155 

- a toast error notification is open when member try to execute a proposal that NOT reach threshold: with error name "ProposalUnderMinumunThreshold"

- a toast error notification is open when user try to execute a proposal already Executed: error name "ProposalNotActive". This is momentary until we fix the subgraph for proposalStatus. Upon fix, the button will be already disabled and the status badge set to "executed"

### CODE REVIEW

File: apps/web/components/Proposals.tsx [hunk 1]
```diff
@@ -251,28 +249,53 @@ export function Proposals({
...
   };
 
-  //test executing a proposal with distribute function
+  //executing proposal distribute function / alert error if not executable / notification if success
   const {
     data: distributeData,
     write: writeDistribute,
     error: errorDistribute,
     isSuccess: isSuccessDistribute,
+    isError: isErrorDistribute,
```
Comment (rperez89) [inline apps/web/components/Proposals.tsx:258]: i am a bit confused with the naming here, which is the difference between error: errorDistribute and then we have another isError: isErrorDistribute? just by the naming seems that is the same maybe we can find a better naming for those

Comment (kamikazebr) [reply apps/web/components/Proposals.tsx:258]: The naming it's just following the pattern, "is" in the beginning indicate that it's boolean, and the other is the error it self. Some checking types get it more clear.

File: apps/web/components/Proposals.tsx [hunk 2]
```diff
@@ -251,28 +250,53 @@ export function Proposals({
...
+    txConfirmationHash: distributeTxConfirmationHash,
+  } = useTransactionNotification(distributeData);
+
+  const {
+    data: waitDistributeData,
+    isSuccess: isWaitDistributeSuccess,
+    status: isWaitDistributeStatus,
+  } = useWaitForTransaction({
+    hash: distributeData?.hash,
+    confirmations: 1,
```
Comment (kamikazebr) [inline apps/web/components/Proposals.tsx:285]: we should be using the CONSTANT here? Good for now, but take notes on it @Mati0x

**Review by kamikazebr — APPROVED** [review]
LGTM

**Pull request merged**